### PR TITLE
[BG-2820] Admin: Validation for free shipping is inconsistent

### PR DESCRIPTION
**Pull request by boris-brtan**
| Q             | A
| ------------- | ---
|Description, reason for the PR| .
|New feature| No 
|BC breaks| No
|Fixes issues| #158
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/src/Resources/views/Admin/Content/TransportAndPayment/freeTransportAndPaymentLimitSetting.html.twig [hunk 1]
```diff
@@ -35,12 +35,15 @@
                     {% set domainName = domain.getDomainConfigById(domainId).getName() %}
                     <tr id="{{ priceLimitForm.vars.id }}" class="js-free-transport-and-payment-price-limit" data-domain-id="{{ domainId }}">
                         {% if isMultidomain() %}
-                            <td>{{ domainName }}</td>
+                            <td>
```
Comment (vitek-rostislav) [inline packages/framework/src/Resources/views/Admin/Content/TransportAndPayment/freeTransportAndPaymentLimitSetting.html.twig]: this was not necessary, at least it does not belong into this commit but never mind :slightly_smiling_face:

File: packages/framework/src/Resources/views/Admin/Content/TransportAndPayment/freeTransportAndPaymentLimitSetting.html.twig [hunk 2]
```diff
@@ -35,12 +35,15 @@
...
+                                {{ domainName }}
+                            </td>
                         {% endif %}
                         <td>
                             {{ form_widget(priceLimitForm.enabled, { attr: { class: 'js-free-transport-and-payment-price-limit-enabled' } } ) }}
                         </td>
-                        <td class="form-inline">
+                        <td class="form-inline in-overlay">
```
Comment (vitek-rostislav) [inline packages/framework/src/Resources/views/Admin/Content/TransportAndPayment/freeTransportAndPaymentLimitSetting.html.twig]: Why? Is this necessary? I have tested it and saw the behavior without the `in-overlay` class but I am not sure if this is a proper solution, maybe you could discuss it with @kulincz

Comment (boris-brtan) [reply packages/framework/src/Resources/views/Admin/Content/TransportAndPayment/freeTransportAndPaymentLimitSetting.html.twig]: seems that without position relative it is outside of the box
image

File: packages/framework/src/Resources/views/Admin/Content/TransportAndPayment/freeTransportAndPaymentLimitSetting.html.twig (hunk 1 above)
Comment (boris-brtan) [reply packages/framework/src/Resources/views/Admin/Content/TransportAndPayment/freeTransportAndPaymentLimitSetting.html.twig]: accepted

Comment (boris-brtan) [reply packages/framework/src/Resources/views/Admin/Content/TransportAndPayment/freeTransportAndPaymentLimitSetting.html.twig]: form_error block was added into free shipping and payment form page        

- to have the same error components in admin forms

**Comment by LukasHeinz** [comment]
BV done, ok for merge

**Pull request merged**